data/actionsmap/yunohost.yml

```diff
@@ -1384,3 +1384,28 @@ hook:
                 -d:
                     full: --chdir
                     help: The directory from where the script will be executed
+
+#############################
+#          Journals         #
+#############################
+journals:
```

Comment:
Because we already uses logs elsewhere and that a log is a continuous stream of output where here we have atomic operations.

I'm not 100% happy with "journal" but it's seems the closest to what I wanted and that wasn't log. I'm open to suggestion (hoping that this doesn't end up in a bikeshding).